main.nf

```diff
@@ -997,8 +1089,8 @@ process megahit {
  */
 
 ch_long_reads_spadeshybrid
-    .combine(ch_short_reads_spadeshybrid, by: 0)
-    .set { ch_short_reads_spadeshybrid }
+    .combine(ch_short_reads_spadeshybrid, by: [0,1])
+    .set { ch_reads_spadeshybrid }
 
 process spadeshybrid {
     tag "$name"
```

Comment:
in case co-assembling is turned on (`params.coassemble_group`), `name` will become `"group$grp"`. Thus if it is performed on a single sample it still uses the sample name

Reply:
Ah alright, missed that!